packages/kyt-core/config/webpack.dev.server.js

```diff
@@ -53,5 +53,6 @@ module.exports = options => ({
   plugins: [
     new webpack.NoErrorsPlugin(),
     new webpack.optimize.LimitChunkCountPlugin({ maxChunks: 1 }),
+    new webpack.BannerPlugin('require("source-map-support").install();', { raw: true, entryOnly: false }),
```

Comment:
I think you need to use the one option argument: 

`{ banner: 'require("source-map-support").install();', raw: true, entryOnly: false }`